FreeRTOS/FreeRTOS
# Use CI-CD-Github-Actions for Spelling and Formatting

Update the CI-CD Workflow files.

Description
-----------
Fix incorrect spelling and formatting on files
Updates to the freertos_demo.yml and freertos_plus_demo.yml workflow files.


Test Steps
-----------
All done in CI-CD

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
